### PR TITLE
chore: update workflows from templates

### PR DESCRIPTION
**Pull request by skjnldsv**
This PR was automatically generated by the workflow dispatcher

### CODE REVIEW

File: .github/workflows/appstore-build-publish.yml [hunk 1]
```diff
@@ -83,7 +83,7 @@ jobs:
         if: steps.check_composer.outputs.files_exists == 'true'
         run: |
           cd ${{ env.APP_NAME }}
-          composer install --no-dev --classmap-authoritative
```
Comment (skjnldsv) [inline .github/workflows/appstore-build-publish.yml:86]: should we have that by default?

File: .github/workflows/phpunit-mysql.yml [hunk 2]
```diff
@@ -100,7 +101,7 @@ jobs:
           DB_PORT: 4444
         run: |
           mkdir data
-          ./occ maintenance:install --verbose --database=mysql --database-name=nextcloud --database-host=127.0.0.1 --database-port=$DB_PORT --database-user=root --database-pass=rootpassword --admin-user admin --admin-pass password
+          ./occ maintenance:install --verbose --database=mysql --database-name=nextcloud --database-host=127.0.0.1 --database-port=$DB_PORT --database-user=root --database-pass=rootpassword --admin-user admin --admin-pass admin
```
Comment (skjnldsv) [inline .github/workflows/phpunit-mysql.yml:104]: ah? I see some repo with admin, the other with password

Comment (skjnldsv) [reply .github/workflows/phpunit-mysql.yml:104]: In the oci workflow it's `admin`

Comment (skjnldsv) [reply .github/workflows/phpunit-mysql.yml:104]: Seems like it doesn't change anything according to the tests passing

File: .github/workflows/appstore-build-publish.yml (hunk 1 above)
Comment (nickvergessen) [reply .github/workflows/appstore-build-publish.yml:86]: I guess so

File: .github/workflows/phpunit-mysql.yml (hunk 2 above)
Comment (nickvergessen) [reply .github/workflows/phpunit-mysql.yml:104]: yeah admin admin is new fixed default.
Talk is running integration tests not from this action, so no result change

File: .github/workflows/dependabot-approve-merge.yml [hunk 3]
```diff
@@ -28,14 +28,13 @@ jobs:
...
-      # GitHub actions bot approve
+      # Github actions bot approve
       - uses: hmarr/auto-approve-action@b40d6c9ed2fa10c9a2749eca7eb004418a705501 # v2
-        if: contains(fromJSON('["/nextcloud/vue-", "/webrtc-adapter-"]'), github.ref) != true
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
       # Nextcloud bot approve and merge request
       - uses: ahmadnassri/action-dependabot-auto-merge@45fc124d949b19b6b8bf6645b6c9d55f4f9ac61a # v2
-        if: contains(fromJSON('["/nextcloud/vue-", "/webrtc-adapter-"]'), github.ref) != true
```
Comment (nickvergessen) [inline .github/workflows/dependabot-approve-merge.yml:39]: custom to Talk and we want to keep those, so webrtc and vue updates don't merge automatically

File: .github/workflows/dependabot-approve-merge.yml [hunk 4]
```diff
@@ -28,14 +28,13 @@ jobs:
       pull-requests: write
 
     steps:
-      # GitHub actions bot approve
+      # Github actions bot approve
       - uses: hmarr/auto-approve-action@b40d6c9ed2fa10c9a2749eca7eb004418a705501 # v2
-        if: contains(fromJSON('["/nextcloud/vue-", "/webrtc-adapter-"]'), github.ref) != true
```
Comment (nickvergessen) [inline .github/workflows/dependabot-approve-merge.yml:33]: same here

File: .github/workflows/update-nextcloud-ocp.yml [hunk 5]
```diff
@@ -17,12 +17,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        branches: ["master", "stable27", "stable26", "stable25", "stable24"]
+        branches: ["main", "master", "stable26", "stable25", "stable24"]
```
Comment (nickvergessen) [inline .github/workflows/update-nextcloud-ocp.yml:20]: missing 27?

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: .github/workflows/update-nextcloud-ocp.yml (hunk 5 above)
Comment (skjnldsv) [reply .github/workflows/update-nextcloud-ocp.yml:20]: wut

Comment (skjnldsv) [reply .github/workflows/update-nextcloud-ocp.yml:20]: Maybe I ran too fast :thinking:

Comment (skjnldsv) [reply .github/workflows/update-nextcloud-ocp.yml:20]: Ah no, this is a different workflow

Comment (skjnldsv) [reply .github/workflows/update-nextcloud-ocp.yml:20]: https://github.com/nextcloud/.github/pull/161

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**